docker-compose.yml

```diff
@@ -85,6 +85,21 @@ services:
     depends_on:
       - base
 
+  crashpad-symbolize:
+    build:
+      args:
+        - GITHUB_TAG=${GITHUB_TAG:-24.lts.30}
+        - ARCHITECTURE=${ARCHITECTURE:-arm64}
+        - SB_API_VERSION=${SB_API_VERSION:-15}
+        - CONFIG=${CONFIG:-qa}
+      context: ./docker/crashpad_symbolize
+      dockerfile: Dockerfile
+    image: crashpad-symbolize
+    volumes:
+      - ${MINIDUMP_PATH}:/root/minidump_directory/minidump.dmp
```

Comment:
Tiny tweak: Is is worth adding a default path value here ? e.g `${MINIDUMP_PATH:minidump.dmp}` - or will the command fail with a clearer error message if it's not set ?